### ISSUE TITLE
Can't change table display length from -1

### ISSUE THREAD

**Issue by dkids** [DobyTang/LazyLibrarian#1397]
Want to see if setting to 100 speeds things up.   Try using pulldown to select 100, press save changes, go to eBooks and wait.   Go back to Config, the value is back to -1.

Repeated with 50.   

Looked at config.ini and it doesn't seem to be a value in there that I can find.

**Comment by philborman** [comment]
You can change table display length "on the fly" from each tables dropdown box top left. The table drawing code remembers the change itself so the config value is really only used at startup to set the initial value. Variable is stored as displaylength in config.ini, but needs tidying up. I might remove it from config and just use the dropdowns, more useful that way if you change it regularly.

**Comment by dkids** [comment]
This seems to be fixed.  I have the length set to 50 and I see displaylength in the config.ini.  I set it to -1 and saved it, then checked config.ini and it was correct.   Set it back to 50 and checked and it's working fine.   

When I first reported it, I did search config.ini for "-1" and didn't find it because I didn't know the variable name.   Not sure what might have changed but things are working fine as far as I'm concerned.
